rust/src/lib.rs

```diff
@@ -1771,14 +1771,14 @@ to_from_bytes!(NativeScript);
 #[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
 pub enum ScriptHashNamespace {
     NativeScript,
-    // TODO: do we need to update this for Plutus?
+    PlutusScript,
 }
 
 #[wasm_bindgen]
 impl NativeScript {
-    pub fn hash(&self, namespace: ScriptHashNamespace) -> ScriptHash {
+    pub fn hash(&self) -> ScriptHash {
         let mut bytes = Vec::with_capacity(self.to_bytes().len() + 1);
-        bytes.extend_from_slice(&vec![namespace as u8]);
+        bytes.extend_from_slice(&vec![ScriptHashNamespace::NativeScript as u8]);
```

Comment:
If we don't define numbers for enum ScriptHashNamespace we can find out problems when order of definitions in ScriptHashNamespace will changed

Reply:
good point

Reply:
See new commit: https://github.com/Emurgo/cardano-serialization-lib/pull/436/commits/aa94d0ca9a4e9932b5b272555ac4fe0184eede65